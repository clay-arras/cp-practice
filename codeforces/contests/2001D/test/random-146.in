1
11
10 15 6 2 7 3 24 9 3 16 1
