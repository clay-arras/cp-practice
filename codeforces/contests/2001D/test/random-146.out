10
10 15 6 2 7 3 24 9 16 1
